Java/Java.sublime-syntax

```diff
@@ -227,7 +338,15 @@ contexts:
                 - match: "}"
                   pop: true
                 - include: class-body
+
   keywords:
+    - match: '::'
```

Comment:
>but I will say that my color scheme (which colors keyword.operator and not punctuation) makes the test example look nicer with a keyword-based scope than a punctuation one.

That's totally subjective though. I'm fine with my Python code not showing dots as blue (for `keyword.operator`).

Either way, this is apparently a discussion about whether or not accessors are in fact "operators". I vote for a `keyword.operator.accessor` scope for all of them, if we were to decide on them being operators.
By the way, semantically this has the same scope as dot access for attributes like `this.field_name`, but I don't think those are scoped as `keyword.operator`, are they?